Nemo11235/CS160-Final-Project
# Winning pop up

Description:
Now when the user guessed the correct word, a pop-up window should show a congrats message.

QA Step:
1. on the home page, enter "apple", which is the correct word;
2. click on enter, then the winning pop up window should show up, like so:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/26106407/161262149-ff68c6b7-8344-477a-9dc1-3f3c9516a0e7.png">
3. click on the "close" button, it should be closed.
